src/components/Tag.jsx

```diff
@@ -1,10 +1,10 @@
 import icX from "../assets/ic_X.svg";
 import styles from "./Tag.module.css";
 
-function Tag({ children }) {
-    return (<div className={styles.tag} >
-        {children}
-        <img src={icX} />
+function Tag({ onXClick, children, className, ...props }) {
+    return (<div className={styles.tag+' '+className+' '+(onXClick ? styles.x : '')} {...props}>
```

Comment:
💊 제안
아래처럼 작성하시는 것도 가능합니다~
```suggestion
    return (
      <div className={`${styles.tag} ${className} ${onXClick ? styles.x : ''}`} {...props}>
```